config/routes.rb

```diff
@@ -10,6 +10,9 @@
     post "/auth/onelogin_identity", to: "omniauth_callbacks#onelogin"
   end
 
+  delete "deauth/onelogin", to: "deauth#onelogin"
+  get "deauth/onelogin/callback", to: "deauth#onelogin_callback"
```

Comment:
Not a blocker / out of scope but if these are nested under the `journey` we get access to the `journey` param in the callback so we know where they came from without relying on the session, which will let us clean up some stuff in the claims controller. Again not a blocker!

Reply:
for one login deauth we don't rely on the session. the session is immediately deleted after we redirect them to one login logout flow. we send one login a `state` param which we receive back which we can therefore use to send them to where they should be.

i'm not sure we really want to nest the auth or deauth endpoints? omniauth typically follows the pattern of
- `/auth/:provider`
- `/auth/:provider/callback`
which i've tried to mimic for deauth.

we will have quite a few of these now and we may want to convert home of these different handlers to independent omniauth strategies we can register and each one knows when to handle the request/callback

i can't really comment on the DSI flows as I've not worked with them close enough to know if they'll work with this pattern?